data/lang/module-secondhand/en.json

```diff
@@ -3,6 +3,22 @@
     "description": "Buy a piece of ship equipment",
     "message": "Buy"
   },
+  "EQUIPMENT_NO_COMPATIBLE_SLOT": {
+    "description": "Message given when the second-hand equipment is not compatible with the ship slots.",
+    "message": "This equipment item is not compatible with your ship."
+  },
+  "EQUIPMENT_NO_FREE_SLOT": {
+    "description": "Message shown by second-hand market if the ship does not have a free slot for the equipment.",
+    "message": "Your ship does not have a free slot for this equipment."
+  },
+  "EQUIPMENT_NO_SPACE": {
+    "description": "Message shown by second-hand market if the equipment requires a free slot or free capacity to fit",
+    "message": "Your ship does not have enough free space for this equipment item."
+  },
+  "EQUIPMENT_SLOT_NO_SPACE":{
+    "description": "Message given when the ship has a compatible slot, but not enough space left to fit the equipment item.",
+    "message": "Your ship has a compatible slot, but not enough space to install this equipment."
+  },
```

Comment:
These strings should be normalized to use only one of "equipment", "equipment item" or "item". Currently it's a 50/50 split between "equipment" and "equipment item" (of which I'd prefer "equipment item" or just "item").